PoissonRegression.ipynb

```diff
@@ -0,0 +1,5371 @@
+{
```

Comment:
<p>&gt; The exponentiated mean of the coefficient <code>b1</code> is roughly half of that of <code>b2</code> which makes sense because in the data that we generated, the number of sneezes was more sensitive to the medicinal intake as compared to the alcohol consumption</p><p><br></p><p>This is a really long sentence. I would add a period at the end of `b2`, and then start the following sentence with "This makes sense because...".</p>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Changed.</p>